scene/resources/world.cpp

```diff
@@ -160,6 +161,19 @@ struct SpatialIndexer {
 			for (int i = 0; i < culled; i++) {
 				//notifiers in frustum
 
+				// check and remove notifiers that have a max range
+				VisibilityNotifier &nt = *ptr[i];
+				if (nt.is_max_distance_active()) {
+					Vector3 offset = nt.get_world_aabb_center() - cam_pos;
```

Comment:
We should likely handle orthogonal cameras here. They should use distance to the camera plane instead of origin otherwise objects will disappear while moving horizontally across the screen.